### PR TITLE
Add googleCloudLog, improve logging of info; alert on failed liquidation

### PR DESCRIPTION
**Pull request by scott-silver**

### CODE REVIEW

File: scripts/liquidation_bot/liquidateUnderwaterBorrowers.ts [hunk 1]
```diff
@@ -22,15 +23,18 @@ async function attemptLiquidation(
...
       pairToken: daiPool.tokenAddress,
       poolFee: daiPool.poolFee
     });
-    console.log(`Successfully liquidated ${targetAddresses}`);
+    googleCloudLog(LogSeverity.INFO, `Successfully liquidated ${targetAddresses} via ${liquidator.address}`);
   } catch (e) {
-    console.log(`Failed to liquidate ${targetAddresses}`);
-    console.log(e.message);
+    googleCloudLog(
+      LogSeverity.ALERT,
```
Comment (kevincheng96) [inline scripts/liquidation_bot/liquidateUnderwaterBorrowers.ts:35]: Does this actually alert someone right now?

Comment (scott-silver) [reply scripts/liquidation_bot/liquidateUnderwaterBorrowers.ts:35]: not yet, but it means we can construct a Policy that alerts if there are more than X number of `ALERT` level logs within a period of time

Comment (kevincheng96) [reply scripts/liquidation_bot/liquidateUnderwaterBorrowers.ts:35]: Nice

**Review by kevincheng96 — APPROVED** [review]
👍

**Pull request merged**